From 10da5225e41d1f7ee360920fa5bac239caa46e9f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hannes=20M=C3=BCller?= <>
Date: Wed, 3 Dec 2025 21:12:56 +0100
Subject: [PATCH] libguile/posix: Fix const warning in CPU_ISSET usage

The CPU_ISSET macro expects a mutable pointer on some systems. Passing
a const pointer triggers a discarded-qualifiers warning.

* libguile/posix.c (cpu_set_to_bitvector): Remove const qualifier.
---
 libguile/posix.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libguile/posix.c b/libguile/posix.c
index 10e6f2849..734d16d84 100644
--- a/libguile/posix.c
+++ b/libguile/posix.c
@@ -2336,7 +2336,7 @@ SCM_DEFINE (scm_setpriority, "setpriority", 3, 0, 0,
 
 #ifdef HAVE_SCHED_GETAFFINITY
 static SCM
-cpu_set_to_bitvector (const cpu_set_t *cs)
+cpu_set_to_bitvector (cpu_set_t *cs)
 {
   SCM bv;
   size_t cpu;
-- 
2.47.3

